src/node/utxo_snapshot.h

```diff
@@ -33,6 +39,30 @@ class SnapshotMetadata
 
     SERIALIZE_METHODS(SnapshotMetadata, obj) { READWRITE(obj.m_base_blockhash, obj.m_coins_count); }
 };
+
+//! The file in the snapshot chainstate dir which stores the base blockhash. This
+//! is needed to reconstruct snapshot chainstates on init.
+const fs::path SNAPSHOT_BLOCKHASH_FILENAME{"base_blockhash"};
```

Comment:
comment can be extended "There should be a single snapshot at any given time"

Reply:
Done.